### DISCUSSION TITLE
Error message in console.log

### DISCUSSION THREAD

**Discussion by myon1664** [add2cal/add-to-calendar-button#174] [Q&A] [answered]
Hi,

I'm getting this message in console.log.

_DevTools failed to load source map: Could not load content for https://cdn.jsdelivr.net/npm/atcb.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE_

Any idea what's wrong?

Regards,
Chrsitian

**Comment by jekuer** [comment, marked as answer]
That's nothing you need to worry about.
The source map is not relevant for anything, except making the code in the console better readable.

The problem comes from jsdeliver not able to fetch the real map url.

We will optimize this soon with a better approach 😊.

**Reply by myon1664** [reply]
Hi jekuer,
Thanks for the quick answer!
And thanks so much for this really nice development :-)
Great design and easy to integrate. I love it.
Kind regards,
Christian
